services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/events/MapMessageCountProgressionEventRepository/MapMessageCountProgressionRepositoryImpl.java

```diff
@@ -0,0 +1,77 @@
+package us.dot.its.jpo.ode.api.accessors.events.MapMessageCountProgressionEventRepository;
+
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.stereotype.Component;
+
+
+import org.springframework.data.domain.Sort;
+
+import us.dot.its.jpo.conflictmonitor.monitor.models.events.MapMessageCountProgressionEvent;
+import us.dot.its.jpo.ode.api.ConflictMonitorApiProperties;
+
+// @TODO: Add rest endpoints to access MessageCountProgressionEvents
```

Comment:
question: is this already handled in here at this point and can be resolved?

Reply:
removed